Data/DataPreferences.swift

```diff
@@ -38,5 +38,8 @@ public extension Preferences {
             public static let previousAttemptedVersion
                 = Option<String?>(key: "database.document-to-support-directory-migration.previous-attempted-version", default: nil)
         }
+        
+        public static let _1_12_1_BoookmarkRestorationCompleted
```

Comment:
Can we rename this to `_Bookmark_v1_12_1RestorationCompleted`?
I don't particularly like variables starting with numbers but I don't know. Up to you.

Reply:
I can do that